WordPressAuthenticator/Signin/LoginPrologueViewController.swift

```diff
@@ -241,22 +251,58 @@ class LoginPrologueViewController: LoginViewController {
 
-        if configuration.enableSiteCreation {
-            let createSiteTitle = displayStrings.siteCreationButtonTitle
-            buttonViewController.setupBottomButton(title: createSiteTitle, isPrimary: false, accessibilityIdentifier: "Prologue Create Site Button", onTap: simplifiedLoginSiteCreationCallback())
+        setButtonViewMargins(forWidth: view.frame.width)
+        let displayStrings = WordPressAuthenticator.shared.displayStrings
+        let buttons: [StackedButton]
+
+        let continueWithWPButton = StackedButton(title: displayStrings.continueWithWPButtonTitle,
+                                                 isPrimary: true,
+                                                 configureBodyFontForTitle: true,
+                                                 accessibilityIdentifier: "Prologue Continue Button",
+                                                 style: primaryButtonStyle,
+                                                 onTap: loginTapCallback())
+        let enterYourSiteAddressButton = StackedButton(title: displayStrings.enterYourSiteAddressButtonTitle,
+                                                       isPrimary: false,
+                                                       configureBodyFontForTitle: true,
+                                                       accessibilityIdentifier: "Prologue Self Hosted Button",
+                                                       style: secondaryButtonStyle,
+                                                       onTap: siteAddressTapCallback())
+        var createSiteButton = StackedButton(title: displayStrings.siteCreationButtonTitle,
+                                             isPrimary: false,
+                                             configureBodyFontForTitle: true,
+                                             accessibilityIdentifier: "Prologue Create Site Button",
+                                             style: secondaryButtonStyle,
+                                             onTap: simplifiedLoginSiteCreationCallback())
+
+        if configuration.enableWPComLoginOnlyInPrologue && configuration.enableSiteCreation {
+            buttons = [continueWithWPButton,
+                       createSiteButton]
+        } else if configuration.enableWPComLoginOnlyInPrologue {
+            buttons = [continueWithWPButton]
+        } else if configuration.enableSiteCreation {
+            createSiteButton.stackView = .bottom
+            buttons = [continueWithWPButton,
+                       enterYourSiteAddressButton,
+                       createSiteButton]
+        } else {
+            DDLogError("Failed to create `StackedButton`s in login progue screen.")
+            buttons = []
         }
 
-        setButtonViewControllerBackground(buttonViewController)
+        let showDivider = configuration.enableWPComLoginOnlyInPrologue == false && configuration.enableSiteCreation == true
```

Comment:
❓ Can we use `orDividerNeededInPrologueButtons` here instead?

Reply:
This is a different condition. I have changed the property name to avoid confusion here. https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/695/commits/d92b4b546494c201567e55509295c8c8b868697f